amqp/subscriber.go

```diff
@@ -28,7 +30,7 @@ type Subscriber interface {
 	SubscribeDeviceUplink(appID, devID string, handler UplinkHandler) error
 	SubscribeAppUplink(appID string, handler UplinkHandler) error
 	SubscribeUplink(handler UplinkHandler) error
-	ConsumeUplink(queue string, handler UplinkHandler) error
+	ConsumeMessages(queue string, uplinkHandler UplinkHandler, eventHandler DeviceEventHandler) error
```

Comment:
A queue can have any number of bindings to any exchange. The sole problem is for the consumer. Their is no guarantee that an uplink message will arrive to the right consumer the first time. 
If we have on type of consumer for uplinks and one for events depending on the number of consumer the message might bounce back a lot of time before going to the right consumer. 

With AMQP queue consumer should all do the same thing and the queue will balance the load. So we either put the charge of routing uplinks and event to AMQP with two queues which is faster but add more configuration etc...Or we have one queue and it's the consumer that call different handler depending on the routing key.